src/tests/functional/plugin/conformance/subgraphs_dumper_new/include/utils/node.hpp

```diff
@@ -167,6 +168,48 @@ inline std::string get_node_type(const std::shared_ptr<ov::Node>& node) {
     return "static";
 }
 
+static std::map<std::string, std::string> get_max_ops_versions() {
+    std::map<std::string, std::set<std::string>> unique_ops = FuncTestUtils::get_unique_ops();
+    std::map<std::string, std::string> max_ops_versions;
+
+    size_t priority = 1;
+    for (auto op_info : unique_ops) {
+        std::vector<int> available_opsets;
+        for (auto opset : op_info.second) {
+            available_opsets.push_back(std::atoi(opset.c_str()));
+        }
+        int max_opset = *std::max_element(available_opsets.begin(), available_opsets.end());
```

Comment:
std::string:
`opset2` > `opset12`

it is incorrect in our case!the simplest way is to extract opset number and convert to size_t

Reply:
I don't got it, I compare versions as int here, but as string it is right way ?